### DISCUSSION TITLE
Error while running test case.

### DISCUSSION THREAD

**Discussion by wuesh** [NCAR/wrf_hydro_nwm_public#740] [Q&A]
hello,
While running the WRF-Hydro v5.2.0 with the "example_case",I encountered the following error.

Note: The following floating-point exceptions are signalling: IEEE_UNDERFLOW_FLAG

I searched for this question on Google and tried to find an answer, but they said it could be ignored and not considered an error. But the model I ran terminated due to this issue, and this issue cannot be ignored.

In this regard, I am a beginner, So I came here to seek help. How should I solve this problem? Looking forward to your reply!
thanks,
wuesh
![屏幕截图 2024-03-08 105459](https://github.com/NCAR/wrf_hydro_nwm_public/assets/162864781/e3c9e597-1c61-41ee-bb75-6a075e0cf0b6)
[diag_hydro.00001.txt](https://github.com/NCAR/wrf_hydro_nwm_public/files/14549218/diag_hydro.00001.txt)
[diag_hydro.00000.txt](https://github.com/NCAR/wrf_hydro_nwm_public/files/14549223/diag_hydro.00000.txt)

**Comment by aubreyd** [comment]
This is not an error. If you check your diag files, you see that the model completed successfully (simulation from 2011-08-26_00:00:00 to 2011-09-02_00:00:00). Why do you think the model simulation failed?

**Comment by y23q** [comment]
Sorry for borthering.While running the WRF-Hydro v5.2.0 with the "example_case",I encountered the following error.

WARNING: get2d_real: failed to find the variables: CHAN_DEPTH and CHAN_DEPTH                                                                                                                                                                                    
 Before read LAKEPARM from NetCDF ./DOMAIN/LAKEPARM.nc
 NLAKES =            1
 start read_route_lake_netcdf
 read_route_lake_netcdf: get_1d_netcdf_real: variable: Dam_Length
 The job is stopped due to the fatal error. get_1d_netcdf
application called MPI_Abort(comm=0x84000004, 1) - process 0
![image](https://github.com/NCAR/wrf_hydro_nwm_public/assets/144974374/4f58fb00-1e05-47bc-a35c-856cdcf04469)

You have been troubled by this question for a long time, how should I solve it?Looking forward to your reply.
